blacklight-cornell/features/catalog_search/book_bags.feature

```diff
@@ -110,7 +110,7 @@ Feature: Book Bags for logged in users
 
 
     @book_bags_cite_selected
-    Scenario Outline: I should be able to view citations for selected items
+    Scenario: I should NOT be able to view citations for selected items
```

Comment:
I get the intent here, but I don't think we should keep tests like this in place long-term. It takes so long to run the entire test suite now that we probably want to eliminate any tests that we don't need. Weigh that against the risk of regression bugs, I suppose.